cmd/server.go

```diff
@@ -40,7 +132,43 @@ func addServerCommand(a *app) *cobra.Command {
 		},
 	}
 
+	cmd.Flags().StringVar(&apiKeyAuthConfig, "api-auth", "", "API-Key authentication credentials")
+	cmd.Flags().StringVar(&basicAuthConfig, "basic-auth", "", "Basic authentication credentials")
+	cmd.Flags().StringVar(&logLevel, "log-level", "info", "Log level")
+	cmd.Flags().StringVar(&logger, "logger", "info", "Logger")
+	cmd.Flags().StringVar(&env, "env", "development", "Convoy environment")
+	cmd.Flags().StringVar(&baseUrl, "base-url", "", "Base Url - Used for the app portal")
+	cmd.Flags().StringVar(&cache, "cache", "redis", "Cache Provider (\"redis\" or \"in-memory\")")
+	cmd.Flags().StringVar(&limiter, "limiter", "redis", "Rate limiter provider (\"redis\" or \"in-memory\")")
```

Comment:
@jirevwe you can use `` for raw string literals, avoids the need for \\ characters

Reply:
Ohh, nice, will try that

Reply:
Nice.